purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/CarouselComponent.kt

```diff
@@ -68,6 +68,9 @@ class CarouselComponent(
     val autoAdvance: AutoAdvancePages? = null,
     @get:JvmSynthetic
     val overrides: List<ComponentOverride<PartialCarouselComponent>> = emptyList(),
+    @get:JvmSynthetic
+    @SerialName("evaluate_unknown_conditions_as")
+    val evaluateUnknownConditionsAs: Boolean? = null,
```

Comment:
I'm unsure about  the position of the property... should it be part of the `condition` itself? As in, we might want to ignore some specific condition if unrecognized, but not another one. It would also be better encapsulated IMO. And if we do need component level support, we can just set the same value for all conditions. Wdyt?